pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/controller/ControllerCommand.java

```diff
@@ -49,7 +61,35 @@
         ClientConfig clientConfig = new ClientConfig();
         clientConfig.register(JacksonJsonProvider.class);
         clientConfig.property("sun.net.http.allowRestrictedHeaders", "true");
-        Client client = ClientBuilder.newClient(clientConfig);
+
+        Client client;
+
+        // If tls parameters are configured, set them in client
+        if (getCLIControllerConfig().isTlsEnabled()) {
+            KeyStore ks = null;
+            try {
+                ks = KeyStore.getInstance("JKS");
+                ks.load(new FileInputStream(new File(getCLIControllerConfig().getTruststore())), null);
+            } catch (KeyStoreException e) {
+                System.out.println("The keystore file is invalid, the keystore type is not supported: " + e.toString());
```

Comment:
Please, use the `output` method that this class inherits instead of `System.out`. Also in other places in this class.

Reply:
Switched to the `output` method instead of using `System.out`.